source/making_payments/index.html.md.erb

```diff
@@ -5,120 +5,139 @@ weight: 70
+</style>
 
-You will receive the ``next_url``  to which you should direct the user to
+<br>
+
+<div class="govuk-warning-text">
+  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+  <strong class="govuk-warning-text__text">
+    <span class="govuk-warning-text__assistive">Warning</span>
+You must not expose the URL with the ``payment_id`` publicly, for example as a URL parameter or in an insecure cookie. You should store it as a secure cookie or in a database.
+  </strong>
+</div>
+
+You will receive the `next_url`  to which you should direct the user to
 complete their payment.
 
-## Tracking the progress of a payment
+## Track the progress of a payment
 
 You can track the progress of a payment while the user is on GOV.UK Pay using
 the <a
 href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/v1/find-payment-by-id"
-target="blank">Find payment by ID</a> call (link opens in new window).
+target="blank">Find payment by ID</a> call [external link]
 
-NOTE: The status of the payment will go through several phases until it either
-succeeds or fails. See the [API reference
-section](/api_reference/#api-reference) for more details.
+The status of the payment will pass through several states until it either
+succeeds or fails. See the [__API reference__
+section](/api_reference/#api-reference) of this documentation for more
+details.
 
-## Choosing the return URL and matching user to payment
+## Choose the return URL and match users to payments
 
 For security reasons, GOV.UK Pay does not add the payment ID or outcome to
-your ``return_url`` as parameters.
+your `return_url` as parameters.
+
+You can match users with payments by using a secure cookie, or creating a
+secure random ID:
+
+### Use a secure cookie
 
-To match up a returning user with their payment, there are two recommended
-methods:
+You can use a secure cookie containing the payment ID from GOV.UK Pay. Your
```

Comment:
I think we want to say “encrypted” cookie here. The term “secure cookie” implies that the cookie is sent over HTTPS and has its `secure` flag set but this still allows the user to read and modify it. We need encryption to stop them from doing that.